src/client/pages/CountryHome/FraHome/hooks/useSections.tsx

```diff
@@ -27,7 +28,11 @@ export const useSections = (): Array<Section> => {
 
     if (!cycle) return null
 
-    sections.push({ name: SectionNames.Country.Home.overview, component: Overview })
+    const showOverview = cycle.published || Areas.isISOCountry(countryIso)
```

Comment:
would it be better to check that it's fra2025 vs published cycle?



```suggestion
const assessmentFra2025 = assessment.props.name = 'fra' && cycle.name = '2025'
const showOverview = assessmentFra2025  || Areas.isISOCountry(countryIso)
```

Reply:
I am not sure if it matters, only case is that is it possible to publish the data before the dashboard?

Reply:
[1521a9d](https://github.com/openforis/fra-platform/pull/3874/commits/1521a9d32dcd024fd342f2abb4d81828d3de104c)